service/raft/group0_state_machine.cc

```diff
@@ -16,6 +16,7 @@
 #include "mutation/async_utils.hh"
 #include "seastar/core/abort_source.hh"
 #include "seastar/core/on_internal_error.hh"
+#include "seastar/coroutine/parallel_for_each.hh"
```

Comment:
Need to use <> includes for seastar

Reply:
Right. copy-pasta from above. I'll fix the adjacent includes too

Reply:
https://github.com/scylladb/scylladb/pull/18513